### PR TITLE
avimetaedit-app 1.0.1

### PR DESCRIPTION
**Pull request by g-maxime**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.

Additionally, if **adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked the cask was not already refused in [closed issues].
- [x] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/caskroom/homebrew-cask/pulls
[closed issues]: https://github.com/caskroom/homebrew-cask/issues?q=is%3Aissue+is%3Aclosed
[the correct repo]: https://github.com/caskroom/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask
[version-checksum]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/sha256.md#updating-the-sha256

### CODE REVIEW

**Comment by miccal** [comment]
https://github.com/Homebrew/homebrew-core/pull/18621

File: Casks/avimetaedit-app.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+cask 'avimetaedit-app' do
```
Comment (commitay) [inline Casks/avimetaedit-app.rb]: Please change this to `avi-metaedit-app`

**Review by commitay — CHANGES_REQUESTED** [review]

**Comment by commitay** [comment]
https://github.com/caskroom/homebrew-cask/pull/39078#issuecomment-332979343

**Comment by g-maxime** [comment]
Renamed, thanks to @commitay

**Comment by miccal** [comment]
@g-maxime thank you for your contribution, but we cannot accept this Cask at this time.

**Pull request closed without merging**